<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
    "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" >
  <head>
    <meta http-equiv="content-type" content="text/html; charset=iso-8859-1" />
    <meta name="author" content="Lior Pachter" />
    <meta name="keywords" content="Computational biology" />
    <meta name="description" content="" />
    <meta name="robots" content="all" />

    <title>UC Berkeley Mathematical and Computational Biology &raquo; Software</title>

    <style type="text/css" media="all">
      @import "stylew.css";
    </style>

  </head>


  <body>
  <div id="container">


    <div id="Header">
<img src="ucbsealsml.gif" alt="UC Berkeley seal" />
<p><span>Lior Pachter's Website</span> <br/>Departments of <a href="http://math.berkeley.edu">Mathematics</a>, <a href="http://mcb.berkeley.edu">Molecular & Cell Biology</a> and <a href="http://www.eecs.berkeley.edu/">Electrical Engineering & Computer Science</a><br></p>

<div id="Menu">
  <ul id="links">
    <li>
      <a href="http://math.berkeley.edu/~lpachter/index.html" title="Home">Home</a>
    </li>
    <li>
      <a href="http://liorpachter.wordpress.com/" title="News">News (blog)</a>
    </li>
    <li>
      <a href="http://math.berkeley.edu/~lpachter/group.html" title="Group">Group</a>
    </li>
    <li>
      <a href="http://math.berkeley.edu/~lpachter/publications.html" title="Publications">Publications</a>
    </li>
    <li>
      <a href="http://pachterlab.github.io/index.html" title="Software">Software</a>
    </li>
    <li>
      <a href="http://math.berkeley.edu/~lpachter/courses.html" title="Courses">Courses</a>
    </li>
<li>
<a href="http://math.berkeley.edu/~lpachter/contact.html" title=Contact">Contact</a>
</li>
    <li>
      <a href="http://math.berkeley.edu/~lpachter/lab.html" title="Lab">Lab</a>
    </li>
  </ul>
</div> <!-- end Menu-->
</div> <!-- end Header-->


    <div id="Content">

<dl>
<dt>Software developed in the Pachter group and still under active development in the group</dt>
<ul>
<li>
<a href="http://pachterlab.github.io/sleuth/">sleuth</a> (2015) RNA-Seq differential analysis
</li>
<li>
<a href="http://pachterlab.github.io/kallisto/">kallisto</a> (2015) RNA-Seq quantification
</li>
<li>
<a href="http://bio.math.berkeley.edu/cgal/">CGAL</a> (2013) Computing genome assembly likelihoods
</li>
</ul>
</dl>

<dl>
<dt>Software developed in the Pachter group but now maintained/developed elsewhere</dt>
<ul>
<li>
<a href="http://bio.math.berkeley.edu/eXpress/">eXpress</a> (2012) Streaming quantification for high-throughput sequencing (now supported by Adam Roberts)</li>
<li><a href="http://www.math.miami.edu/~vhower/ReadSpy.html">ReadSpy</a> (2012) Assessment of uniformity in RNA-Seq reads (now supported by <a href="http://www.math.miami.edu/~vhower/">Valerie Hower</a> and her group at the University of Miami) </li>
<li>
<a href="http://bio.math.berkeley.edu/SysCall/">SysCall</a> (2011) Distinguishing heterozygous sites from systematic error in high-thoughput sequenced reads (now supported by <a href="http://math.mcb.berkeley.edu/~meromit/">Meromit Singer</a> at the Broad Institute)
</li>
<li>
<a href="http://www.cs.berkeley.edu/~meromit/MetMap.html">MetMap</a> (2010) Analysis of Methyl-Seq experiments (now supported by <a href="http://math.mcb.berkeley.edu/~meromit/">Meromit Singer</a> at the Broad Institute)
</li>
<li>
<a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a> (2010) Transcript assembly and abundance estimation for RNA-Seq (now supported by <a href="http://cole-trapnell-lab.github.io">Cole Trapnell</a> and his group at the University of Washington)
</li>
<li><a href="http://tophat.cbcb.umd.edu/">TopHat</a> (2009) Splice junction mapper for short RNA-seq reads (now supported by <a href="http://bioinformatics.igm.jhmi.edu/salzberg/Salzberg/Salzberg_Lab_Home.html">Steven Salzberg</a> and his group at Johns Hopkins University)</li> 
<li>
<a href="http://orangutan.math.berkeley.edu/fsa/">FSA</a> (2009) <b>F</b>ast <b>S</b>tatistical <b>A</b>lignment (now supported by <a href="http://labs.fhcrc.org/bradleyr/">Robert Bradley</a> and his group at FHCRC)
</li>
<li>
<a href="http://www.biostat.wisc.edu/~cdewey/mercator/">MERCATOR</a> (2004) Homology mapping (now supported by <a href="http://www.biostat.wisc.edu/~cdewey/">Colin Dewey</a> and his group at the University of Wisconsin)
</li>
<li>
<a href="http://genome.lbl.gov/vista/mvista/about.shtml">VISTA</a> (2000) Visualization tool for global alignments
 (now supported by <a href="http://www.jgi.doe.gov/whoweare/dubchak.html">Inna Dubchak</a> and her group  at the JGI) </li>
</ul>
</dl>


<dl>
<dt>Retired Software</dt>
These programs, originally developed in the Pachter group, are no longer under active development and are not being supported.
<ul>
<li>
<u>AMAP</u> (2007) Protein multiple alignment (recommended instead: <a href="http://orangutan.math.berkeley.edu/fsa/">FSA</a>)
</li>
<li>
<u>GENEMAPPER</u> (2006) Reference based gene annotation (recommended instead: <a href="http://www.nature.com/nbt/journal/v28/n5/abs/nbt.1621.html"> an RNA-Seq experiment</a>)
</li>
<li>
<u>MJOIN</u> (2006) Neighbor joining with subtree weights (archived <a href="http://bio.math.berkeley.edu/mjoin/">here</a>)
</li>
<li>
<u>PARALIGN</u> (2006) Alignment polytope construction (archived <a href="http://bio.math.berkeley.edu//paralign/">here</a>) 
</li>
<li>
<u>SLIM</u> (2003) Minimum network design for optimizing the search space for pair hidden Markov models (archived <a href="http://bio.math.berkeley.edu/slim/">here</a>) </li>
<li>
<u>SLAM</u> (2003) Pairwise simultaneous alignment and gene finding (recommended instead: <a href="http://www.nature.com/nbt/journal/v28/n5/abs/nbt.1621.html">an RNA-Seq experiment</a>)
</li>
<li>
<u>MAVID</u> (2003) Multiple alignment of large genomic sequences (recommended instead: <a href="http://orangutan.math.berkeley.edu/fsa/">FSA</a>)
</li>
</ul>
</dl>

    </div>	

    <div id="Picture">
    </div>

    <div id="Footer">

    </div>
</div>

</body>
</html>
